src/EditorFeatures/Core/LanguageServer/Handlers/Completion/CompletionHandler.cs

```diff
@@ -185,42 +185,6 @@ bool IsValidTriggerCharacterForDocument(Document document, char triggerCharacter
                 TextSpan? defaultSpan,
                 LSP.Range? defaultRange,
                 CancellationToken cancellationToken)
-            {
-                if (supportsVSExtensions)
```

Comment:
this method was kind of useless, so I just merged it with the local `CreateCompletionItemAsync`